kubernetes/src/test/java/oracle/kubernetes/operator/helm/HelmOperatorValues.java

```diff
@@ -100,6 +119,17 @@ private void addDomainsNamespaces(HashMap<String, Object> map) {
     }
   }
 
+  private void addOperatorImagePullSecrets(HashMap<String, Object> map) {
+    String secretName = getWeblogicOperatorImagePullSecretName();
+    if (secretName != null && (!secretName.isEmpty())) {
+      Map<String, String> secret = new HashMap();
+      secret.put("name", secretName);
+      List<Map<String, String>> secrets = new ArrayList();
+      secrets.add(secret);
+      map.put("operatorImagePullSecrets", secrets);
+    }
```

Comment:
Consider using the built-ins to handle singleton lists like this. 

map.put("operatorImagePullSecrets", 
           Collections.singletonList(
                       Collections.singletonMap("name", secretName)));

is easier to understand at a glance, especially for something like a map of lists of maps.